### PR TITLE
Consolidate "Using Rudr" content across readme, concepts

### LINKED ISSUE #414: Discussion: "Using Rudr" needs to be shorter and scenario based
Description of the Issue: 

- Its not immediately clear what I'm achieving here in the [Using Rudr](https://github.com/oam-dev/rudr/blob/master/docs/concepts/using_rudr.md) concept article. 

One approach to a solution:

- We could rework it to be based on a real world scenario with a clear goal.

### PR DESCRIPTION
**Pull request by erikadoyle**
Here's an attempt to make the docs landing page succinctly describe the Rudr value prop and concepts.
 
- Moved @macolso's awesome scenario-based comics into their respective concept guides to make landing TOC more concise. Also consolidated "images" folder into "media".
- Ensured all the goodness from original "Using Rudr" guide is accounted for on landing page and/or in concepts guides (I actually used *Using Rudr* as the base material for all the concept guides)
- Tried to follow bluey-purple color scheme from https://oam.dev/, but its easy enough to change any aspect of the images as needed, so let me know. The source is in powerpoint (thanks @suhuruli for the base graphics!)

/cc @suhuruli, @macolso 

Fixes #414, #438

### CODE REVIEW

**Comment by suhuruli** [comment]
me as well :)

**Comment by vturecek** [comment]
Agreed, this looks **really** good.

File: docs/README.md [hunk 1]
```diff
@@ -1,50 +1,91 @@
...
+
+To get started, check out the *Install Rudr* guide and the *Tutorial*. The *Concepts* guides will walk you through all the available options Rudr provides to run and manage applications, and the *How-Tos* cover more advanced scenarios.
+
 ## [Install Rudr](./setup/install.md)
 Install the Rudr runtime and its dependencies.
 
+## [Tutorial](./tutorials/deploy_and_update.md)
+Learn how to deploy, inspect, and update a Rudr application.
+
 ## Concepts
```
Comment (vturecek) [inline docs/README.md]: Small nit - only thing that looks a little bit off is that "Install Rudr" and "Tutorial" are links with the same heading size as Concepts, which is a separate section. 

Maybe we could put the "Install Rudr" and "Tutorial" links under a section called "Getting Started" so it looks more like the Concepts section?

File: docs/README.md [hunk 2]
```diff
@@ -1,50 +1,93 @@
...
+*Rudr* is the Kubernetes reference implementation of the [Open Application Model](https://github.com/oam-dev/spec) (OAM) specification, a team-centric standard for building cloud-native apps, where:
+
+<table border=0 rules=none>
+<tr>
+<td><img alt="Developer role" src="./media/developer-role.png" /></td>
+<td><b>Developers</b> define application <b>components</b>,</td>
+</tr>
+<tr>
+<td><img alt="Application operator role" src="./media/app-operator-role.png" /></td>
+<td><b>Application operators</b> create instances of those components and assign them to <b>application configurations</b>, and</td>
```
Comment (resouer) [inline docs/README.md]: do we also need to mention trait (or operational capabilities) here?

**Comment by hongchaodeng** [comment]
lgtm
This makes the doc look exciting!

**Pull request merged**